doc/devel/contributing.rst

```diff
@@ -89,13 +89,13 @@ use on a preexisting environment!
 
 A new environment can be set up with ::
 
-   python3 -mvenv /path/to/devel/env
+   python3 -mvenv <file folder location>
 
 and activated with one of the following::
 
-   source /path/to/devel/env/bin/activate  # Linux/macOS
-   /path/to/devel/env/Scripts/activate.bat  # Windows cmd.exe
-   /path/to/devel/env/Scripts/Activate.ps1  # Windows PowerShell
+   source <file folder location>/activate  # Linux/macOS
```

Comment:
this needs the `bin/` or it won't work I think

Reply:
Ah, yes. That addition is necessary for the suggested change. Thank you!